src/error.rs

```diff
@@ -17,7 +17,17 @@ pub enum ExternalError {
 /// The error type for when the requested operation is not supported by the backend.
 #[derive(Clone)]
 pub struct NotSupportedError {
-    _marker: (),
+    inner: NotSupportedRepr,
```

Comment:
Shouldn't we do it uniformly with the rest of the errors? Since we generally point to `platform_impl` from here?

Though, in the current state of things it doesn't matter that much.

Reply:
That would add a lot of additional code for little benefit